clients/hadoopfs/pom.xml

```diff
@@ -276,8 +276,8 @@ To export to S3:
     <dependencies>
         <dependency>
             <groupId>io.lakefs</groupId>
-            <artifactId>api-client</artifactId>
-            <version>0.91.0</version>
+            <artifactId>sdk-client</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
```

Comment:
I wonder if this was supposed to be checked but probably better to find out

Reply:
This is unfortunately needed until I push a first version of the Java client SDK.